root/maptick_plot.gnuplot

```diff
@@ -41,24 +48,25 @@ set yrange [0:]
 set grid ytics
 
 #CSV file is formatted like this
-# id, maptick_to_player, maptick, players, td, td_to_player
-# followed by three entries for everything but the id. avg, min, and max
+# id, maptick_to_player, maptick, players, td, td_to_player, sendmaps_cost_per_tick, per_client, look_for_movable_changes, check_turf_vis_conts, check_hud/image_vis_contents, turfs_in_range, obj_changes, mob_changes, movables_examined
+# followed by three entries for everything but the id: the avg, the min, and the max
 
 #To find the index of the header you want, use this forumula
-# 6 + ((spot in list - 2) * 3) + (spot in list - 1) * header you want
+# 15 + ((spot in list - 2) * 3) + (spot in list - 1) * header you want
+#I HAVE NO IDEA HOW THESE ARE DERIVED REEEEEEEEEEEEEE
 
```

Comment:
FOOOL
I only barely remember

if you think of the csv file as 
```
averagables = (maptick to player, maptick, players, td, td to player)
headers = id + averagables + averagables * (avg, min, max)
```
so each averagable gets an avg, min, and max header.

Then say trying to get the index of avg td, we get 
6 //non averagable header count 
+ (position you're looking for - 2) * 3 //averagable types, the - 2 is to start at 0 plus counteracting id's index 
+ (position you're looking for - 1) * header //header is 1 for avg, 2 for min, 3 for max, same story here really

We're basically indexing a list based on the thing we want, and the "type" of the thing we want (avg, min, max)
It's cursed, and I'm sorry